src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimContinuingOnlineApplicantForSpecNotificationHandler.java

```diff
@@ -79,14 +85,33 @@ private CallbackResponse notifyApplicantSolicitorForClaimContinuingOnline(Callba
 
     @Override
     public Map<String, String> addProperties(CaseData caseData) {
-        return Map.of(
-            CLAIM_LEGAL_ORG_NAME_SPEC, getApplicantLegalOrganizationName(caseData.getApplicant1OrganisationPolicy()
-                .getOrganisation().getOrganisationID(), caseData),
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            ISSUED_ON, formatLocalDate(caseData.getIssueDate(), DATE),
-            RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1()),
-            RESPONSE_DEADLINE, formatLocalDateTime(caseData.getRespondent1ResponseDeadline(), DATE_TIME_AT)
-        );
+
+        if (getMultiPartyScenario(caseData).equals(ONE_V_TWO_TWO_LEGAL_REP)) {
+            return Map.of(
```

Comment:
In the return statement only difference in the Map is the respondent name, other things are same.
Could we create the map before add the common fields and then add names based on the condition.
Then we can return the final map.

Reply:
Unfortunately the multiparty gov notify template contains a different name for the Defendant One Name:
if its 1v1: RESPONDENT_NAME / "defendantName"
if its 1v2: RESPONDENT_ONE_NAME / "defendantOneName"and RESPONDENT_TWO_NAME / "defendantTwoName"